### DISCUSSION TITLE
antd 5 css not supporting in Safari version 13

### DISCUSSION THREAD

**Discussion by bibinantony98** [ant-design/ant-design#43961] [Q&A]
antd 5 css not supported in Safari version 13

`:where` is not supported in Safari browser version 13

I added StyleProvider hashPriority="high" as in documents, but still, antd models are in ` :where`

![Screenshot 2023-08-02 at 1 03 23 PM](https://github.com/ant-design/ant-design/assets/48233705/7d200e3f-0541-4b9d-a42f-413c6041607b)

**Comment by yoyo837** [comment]
https://ant.design/docs/react/compatible-style
